client/src/questions-n-answers/qNa.jsx

```diff
@@ -326,6 +463,19 @@ class QuestionsNAnswers extends React.Component {
    })
 
 
+ }
+
+ addToReported(e, ansId) {
```

Comment:
Not sure if this is a relic of building out addToReported, but seems odd to have it twice, once with event only for logging and the other time setting state